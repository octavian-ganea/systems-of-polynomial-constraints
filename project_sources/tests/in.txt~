% Test 1:
x^2 + y^2 + z^2 - x*y - y*z - z*x  = 0
% (x-y)^2 + (y-z)^2 + (z-x)^2 = 0
x - 2 > 0
y - 2.4 < 0