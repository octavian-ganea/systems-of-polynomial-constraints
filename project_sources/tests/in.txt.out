Parsing file in.txt
Parsing succeded. Results:
List of all variables:
Vector(x, y, z)
List of all polynomials:
Vector((-1.0 y + (-1.0 z))x + (1.0 y^2 + (1.0 z^2) + (-1.0 z)y) + 1.0 x^2, -2.0  + 1.0 x, -2.4  + 1.0 y)

List of desired signs : Vector(=, >, <)
Analysed polynomials: 
(-1.0 y + (-1.0 z))x + (1.0 y^2 + (1.0 z^2) + (-1.0 z)y) + 1.0 x^2
-2.0  + 1.0 x
-2.4  + 1.0 y
vars: Vector(x, y, z)

Univariate projection (on real axis) contains these polynomials: 
-1.0 z
1.0 z^2
-6.0 z
3.0 z^2
-1.0 z + -2.0 
1.0 z^2 + -2.0 z + 4.0 
1.0 z^2 + -2.0 z
3.0 z^2 + -12.0 z + 12.0 
3.0 z^2 + -12.0 z + -4.0 
-2.0 z^2
-2.4 z + 1.0 z^2
5.76  + -2.4 z + 1.0 z^2
-2.4  + -1.0 z
-3.0 z^3
27.0 z^4
9.0 z^4
1.0 z^4
3.0 z
9.0 z^2
12.0 z^2
-3.0 z^2 + 14.399999999999999 z
3.0 z^2 + -14.399999999999999 z + 17.28 
1.0 z^4 + -4.0 z^3 + 4.0 z^2
16.0  + 12.0 z^2 + -24.0 z
-4.0 z^2 + 4.0 z
16.0  + 1.0 z^4 + 16.0 z^2 + -24.0 z
16.0  + 1.0 z^4 + -4.0 z^3 + 12.0 z^2 + -16.0 z
3.0 z^2 + 4.0 
2.0 z^2 + 4.0 
1.0 z^2 + -4.4 z + 4.96 
-1.0 z^2 + 4.4 z + 0.7999999999999998 
27.0 z^4 + -108.0 z^3 + 144.0 z^2
144.0  + -72.0 z^3 + 216.0 z^2 + -288.0 z + 9.0 z^4
-3.0 z + 6.0 
3.0 z^3 + -18.0 z^2 + 24.0 z
48.0  + 48.0 z^2 + -96.0 z
1.0 z^4 + -4.0 z^3
-4.0 z^2
1.0 z^4 + 12.0 z^2
2.0 z^2
1.0 z^2 + -4.4 z + 0.96 
-1.0 z^2 + 4.4 z + 4.8 
27.0 z^4 + -108.0 z^3
-72.0 z^3 + 144.0 z^2 + 9.0 z^4
3.0 z^3 + -18.0 z^2
48.0 z^2
8.0 z^2 + -32.0 z + 1.0 z^4 + -4.0 z^3
16.0  + 1.0 z^4 + -4.0 z^3 + 8.0 z^2 + 16.0 z
-4.0 z + -8.0 
5.76 z^2 + -4.8 z^3 + 1.0 z^4
33.1776  + 17.28 z^2 + -27.648 z + -4.8 z^3 + 1.0 z^4
4.8  + -1.0 z
5.76 z + -4.8 z^2
33.1776  + -27.648 z + 23.04 z^2
5.76  + 4.8 z + 1.0 z^2
5.76  + 4.8 z
207.35999999999999 z^2 + -86.39999999999999 z^3 + 9.0 z^4
298.59839999999997  + 311.04 z^2 + -497.66399999999993 z + -86.39999999999999 z^3 + 9.0 z^4
14.399999999999999  + -6.0 z
34.56 z + -14.399999999999999 z^2
99.53280000000001  + -165.888 z + 69.11999999999999 z^2
24.601599999999998  + 29.28 z^2 + -43.647999999999996 z + -8.8 z^3 + 1.0 z^4
2.8  + -1.0 z
0.6400000000000006  + 17.759999999999998 z^2 + 7.039999999999999 z + -8.8 z^3 + 1.0 z^4
70.04159999999999  + -73.728 z + 23.04 z^2
15.36 z + -7.68  + -4.8 z^2
0.921599999999998  + 21.28 z^2 + -8.44799999999999 z + -8.8 z^3 + 1.0 z^4
23.04  + 9.76 z^2 + 42.239999999999995 z + -8.8 z^3 + 1.0 z^4
-22.1184  + -73.728 z + 23.04 z^2
15.36 z + 11.52  + -4.8 z^2

There are 1559 cells
====== FINAL RESULT FOR in.txt ================================
Map(x -> 2.1999988447755, y -> 2.1999994227709694, z -> 2.199999999881034)
====================================================

